code/Controllers/CMSMain.php

```diff
@@ -223,19 +233,23 @@ public function getResponseNegotiator(): PjaxResponseNegotiator
      * Get pages listing area
      *
      * @return DBHTMLText
+     * @deprecated 5.4.0 Will be renamed to RecordList
      */
     public function PageList()
     {
+        Deprecation::noticeWithNoReplacment('5.4.0', 'Will be renamed to RecordList');
         return $this->renderWith($this->getTemplatesWithSuffix('_PageList'));
```

Comment:
Note I didn't add an implementation for `RecordList()` because that will be using a renamed template, so people would update their calls to the method but not the template in 5 and then still have to make an update in 6 anyway.

It's extremely unlikely anyone will be calling this anyway so kinda moot.